packages/ab-testing/__tests__/index.test.ts

```diff
@@ -201,7 +234,7 @@ describe('ab-testing module', () => {
                 config,
                 1,
                 hashObject({ user_id: 1, email_domain: 'example.com' }, config.salt)
-            ).getCohort('experiment_3')
+            ).getCohort('experiment_any')
```

Comment:
`experiment_3` is not existed. I think its better to change name. I want an idea if you have.